packages/sitecore-jss-react-forms/src/components/form.tsx

```diff
@@ -96,7 +97,11 @@ export class Form extends Component<FormProps, FormState & FieldStateCollection>
       return <div>Form data invalid. Forget to set the rendering contents resolver?</div>;
     }
 
-    const action = `${this.props.sitecoreApiHost}/api/jss/formbuilder?fxb.FormItemId=${form.metadata.itemId}&fxb.HtmlPrefix=${form.htmlPrefix}&sc_apikey=${this.props.sitecoreApiKey}&sc_itemid=${form.contextItemId}`;
+    const action = `${this.props.sitecoreApiHost}/api/jss/formbuilder?fxb.FormItemId=${
+      form.metadata.itemId
+    }&fxb.HtmlPrefix=${form.htmlPrefix}&sc_apikey=${this.props.sitecoreApiKey}&sc_itemid=${
+      form.contextItemId
+    }&sc_lang=${this.props.language || ''}`;
```

Comment:
Should we exclude the query string parm completely if `props.language` is undefined? Not sure if there is any adverse side-effects with sending an empty value.

Reply:
I think that I tested and it was working when the value is an empty string, but for me it makes sense just to remove it